src/config-schema.ts

```diff
@@ -41,4 +41,22 @@ export const configSchema = {
     _default: true,
     _description: 'Whether to show a warning about possibly losing data in the forms dashboard',
   },
+  dataTypeToRenderingMap: {
+    _description: 'A map used to match concept datatypes to rendering types',
+    _type: Type.Object,
+    _default: {
+      Numeric: ['number', 'fixed-value'],
+      Coded: ['select', 'checkbox', 'radio', 'toggle', 'content-switcher', 'fixed-value'],
+      Text: ['text', 'textarea', 'fixed-value'],
+      Date: ['date', 'fixed-value'],
+      Datetime: ['datetime', 'fixed-value'],
+      Boolean: ['toggle', 'select', 'radio', 'content-switcher', 'fixed-value'],
+      Rule: ['repeating', 'group'],
+    },
+  },
```

Comment:
There is a bit technical debt here and might be addressed in a separate PR, but we may need to update this map to include more rendering types that may be supported by the various concept datatypes.